test/Typewriter.Test/Typewriter.Test.csproj

```diff
@@ -79,43 +29,324 @@
     <Content Include="Metadata\MetadataMultipleNamespaces.xml">
       <CopyToOutputDirectory>Always</CopyToOutputDirectory>
     </Content>
-    <Compile Include="Given_a_valid_metadata_file_to_metadata_preprocessor.cs" />
-    <Compile Include="MultipleNamespacesTestRunner.cs" />
-    <Compile Include="Properties\AssemblyInfo.cs" />
-    <Compile Include="Properties\Resources.Designer.cs">
+    <Compile Update="Properties\Resources.Designer.cs">
       <AutoGen>True</AutoGen>
       <DesignTime>True</DesignTime>
       <DependentUpon>Resources.resx</DependentUpon>
     </Compile>
-    <Compile Include="Given_a_valid_metadata_file_to_Typewriter.cs" />
-  </ItemGroup>
-  <ItemGroup>
-    <None Include="app.config" />
-    <None Include="packages.config" />
   </ItemGroup>
   <ItemGroup>
     <Content Include="Resources\dirtyMetadata.xml" />
   </ItemGroup>
   <ItemGroup>
-    <EmbeddedResource Include="Properties\Resources.resx">
+    <EmbeddedResource Update="Properties\Resources.resx">
       <Generator>ResXFileCodeGenerator</Generator>
       <LastGenOutput>Resources.Designer.cs</LastGenOutput>
     </EmbeddedResource>
   </ItemGroup>
   <ItemGroup>
-    <ProjectReference Include="..\..\src\Typewriter\Typewriter.csproj">
-      <Project>{ca08f5d7-aa00-462e-be5e-e4f54c257fa5}</Project>
-      <Name>Typewriter</Name>
-    </ProjectReference>
+    <ProjectReference Include="..\..\src\Typewriter\Typewriter.csproj" />
+  </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="NUnit">
+      <Version>3.12.0</Version>
+    </PackageReference>
+    <PackageReference Include="NUnit.ApplicationDomain">
+      <Version>11.1.0</Version>
+    </PackageReference>
+    <PackageReference Include="NUnit3TestAdapter">
+      <Version>3.17.0</Version>
+    </PackageReference>
+  </ItemGroup>
+  <ItemGroup>
+    <Compile Remove="TestDataCSharp\com\Microsoft\Graph2\CallRecords\Model\CallRecord.cs" />
```

Comment:
I assume this allows file name patterns, so we can use `TestDataCSharp\**\*.cs`

Reply:
fixed, thanks for the suggestion